spec/controllers/submissions_controller_spec.rb

```diff
@@ -103,14 +103,60 @@
     end
 
     it 'should be able to add and access files' do
-      file1 = fixture_file_upload('Shapes.java', 'text/java')
-      file2 = fixture_file_upload('TestShapes.java', 'text/java')
-
       expect(@student).to have_accepted_grouping_for(@assignment.id)
+
+      # Check MIME type for a correct file
```

Comment:
You should create new tests rather than add to existing tests. Define different unit tests for each test input.